### ISSUE TITLE
Error running pc init

### ISSUE THREAD

**Issue by BokiMC** [pynecone-io/pynecone#93]
──────────────────────────────────────────────── Starting Pynecone App ────────────────────────────────────────────────
─────────────────────────────────────────────── Installing Dependencies ───────────────────────────────────────────────
Traceback (most recent call last):
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\Scripts\pc.exe\__main__.py", line 7, in <module>
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\typer\main.py", line 214, in __call__
    return get_command(self)(*args, **kwargs)
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\typer\main.py", line 532, in wrapper
    return callback(**use_params)  # type: ignore
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\pynecone\pc.py", line 78, in run
    frontend_cmd(app)
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\pynecone\utils.py", line 363, in run_frontend
    setup_frontend(app)
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\pynecone\utils.py", line 345, in setup_frontend
    install_dependencies()
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\pynecone\utils.py", line 320, in install_dependencies
    subprocess.call([get_bun_path(), "install"], cwd=constants.WEB_DIR, stdout=PIPE)
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\subprocess.py", line 345, in call
    with Popen(*popenargs, **kwargs) as p:
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\subprocess.py", line 966, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\subprocess.py", line 1435, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified

I have just installed pynecone,i have py and node,everything seems fine on my end

**Comment by spider-z3r0** [comment]
I'm also having exactly the same issue, with the same error. Installed pyncone and node.js about an hour ago. 
Python 3.11.0
Windows 10
VS code

**Comment by FHU-yezi** [comment]
Please use code block ( wrap your terminal outpur with \`\`\` ) for better readability.

**Comment by BokiMC** [comment]
> 

```──────────────────────────────────────────────── Starting Pynecone App ────────────────────────────────────────────────
─────────────────────────────────────────────── Installing Dependencies ───────────────────────────────────────────────
Traceback (most recent call last):
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\Scripts\pc.exe\__main__.py", line 7, in <module>
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\typer\main.py", line 214, in __call__
    return get_command(self)(*args, **kwargs)
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\typer\main.py", line 532, in wrapper
    return callback(**use_params)  # type: ignore
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\pynecone\pc.py", line 78, in run
    frontend_cmd(app)
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\pynecone\utils.py", line 363, in run_frontend
    setup_frontend(app)
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\pynecone\utils.py", line 345, in setup_frontend
    install_dependencies()
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\pynecone\utils.py", line 320, in install_dependencies
    subprocess.call([get_bun_path(), "install"], cwd=constants.WEB_DIR, stdout=PIPE)
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\subprocess.py", line 345, in call
    with Popen(*popenargs, **kwargs) as p:
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\subprocess.py", line 966, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\subprocess.py", line 1435, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified```

**Comment by Alek99** [comment]
> ──────────────────────────────────────────────── Starting Pynecone App ──────────────────────────────────────────────── ─────────────────────────────────────────────── Installing Dependencies ─────────────────────────────────────────────── Traceback (most recent call last): File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main return _run_code(code, main_globals, None, File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code exec(code, run_globals) File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\Scripts\pc.exe__main__.py", line 7, in File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\typer\main.py", line 214, in **call** return get_command(self)(*args, **kwargs) File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1130, in **call** return self.main(*args, **kwargs) File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1055, in main rv = self.invoke(ctx) File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1657, in invoke return _process_result(sub_ctx.command.invoke(sub_ctx)) File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 1404, in invoke return ctx.invoke(self.callback, **ctx.params) File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\click\core.py", line 760, in invoke return __callback(*args, **kwargs) File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\typer\main.py", line 532, in wrapper return callback(**use_params) # type: ignore File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\pynecone\pc.py", line 78, in run frontend_cmd(app) File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\pynecone\utils.py", line 363, in run_frontend setup_frontend(app) File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\pynecone\utils.py", line 345, in setup_frontend install_dependencies() File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\site-packages\pynecone\utils.py", line 320, in install_dependencies subprocess.call([get_bun_path(), "install"], cwd=constants.WEB_DIR, stdout=PIPE) File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\subprocess.py", line 345, in call with Popen(*popenargs, **kwargs) as p: File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\subprocess.py", line 966, in **init** self._execute_child(args, executable, preexec_fn, close_fds, File "C:\Users\bojan\AppData\Local\Programs\Python\Python310\lib\subprocess.py", line 1435, in _execute_child hp, ht, pid, tid = _winapi.CreateProcess(executable, args, FileNotFoundError: [WinError 2] The system cannot find the file specified
> 
> I have just installed pynecone,i have py and node,everything seems fine on my end

Are you running on windows if so see https://github.com/pynecone-io/pynecone/issues/51 we are fixing this

**Comment by Alek99** [comment]
We are trying to get this fix done by our next release this Friday. Thanks for point it out

**Comment by BokiMC** [comment]
> We are trying to get this fix done by our next release this Friday. Thanks for point it out

any temporary fix?Maybe downgrade version or something?I need it asap cuz of school projects etc etc
Thanks!

**Comment by Alek99** [comment]
Ok we'll try and prioritize this issue first, you could run it on a linux window subsystem and it should work.

**Comment by FHU-yezi** [comment]
> Ok we'll try and prioritize this issue first, you could run it on a linux window subsystem and it should work.

WSL, Windows Subsystem of Linux, it requires you to enable Hyper-V, and it has some weak in IO performance.

But it is work, that's enough.

**Comment by FHU-yezi** [comment]
> > We are trying to get this fix done by our next release this Friday. Thanks for point it out
> 
> any temporary fix?Maybe downgrade version or something?I need it asap cuz of school projects etc etc
> Thanks!

We just merged #96 to use npm instead of bun on Windows, you can clone this repo and build your own version to test if this work for your case.

Maybe the production mode still not work on it, we are working on .

**Comment by Alek99** [comment]
> > > We are trying to get this fix done by our next release this Friday. Thanks for point it out
> > 
> > 
> > any temporary fix?Maybe downgrade version or something?I need it asap cuz of school projects etc etc
> > Thanks!
> 
> We just merged #96 to use npm instead of bun on Windows, you can clone this repo and build your own version to test if it works for your case.
> 
> Maybe the production mode still can't be used on Windows, we are working on #97.

Thanks for updating @FHU-yezi

**Comment by Alek99** [comment]
This is fixed and will be included in the 0.1.9 release https://github.com/pynecone-io/pynecone/pull/96
